ros_ws/src/autonomous/emergency_stop/launch/emergency_stop.launch

```diff
@@ -1,14 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <launch>
-<!-- Launches all necessary nodes for the emergency stop -->
+  <!-- Launches all necessary nodes for the emergency stop -->
 
-    <!-- emergency_stop node -->
-    <node
-      respawn="true"
-      pkg="emergency_stop"
-      type="emergency_stop"
-      name="emergency_stop"
-      output="screen" >
-    </node>
+  <!-- emergency_stop node -->
+  <node required="true" pkg="emergency_stop" type="emergency_stop" name="emergency_stop" output="screen">
+  </node>
```

Comment:
Since the node tag has no children, you don't need the closing tag. `<node ... />`
There are more cases of this below.